bin/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This change is so that when testing locally, the script uses the user's preferred bash (such as the up-to-date one installed by homebrew) rather than system bash (which for the case of OS X is bash v3 which doesn't support the `-v` syntax below).